docs/design_spec.md

```diff
@@ -0,0 +1,94 @@
+# Nimbess Agent Design
+
+## Overview
+
+Nimbess Agent's main purpose is to manage an external data plane. The Agent's main responsibilities
+include:
+
+ 1. Translating Northbound (NB) application-specific resources into network oriented resources across the
```

Comment:
it more than application-specific resources? Or is the intention to be more than networking resources?

Reply:
I'm not sure what you mean, but maybe you are saying it should be platform-specific resources into network oriented resources?

Reply:
at the end we are handling just network resources, but application-specific is much more broad than that. Like an app might have some memory or cpu resource it wants, but wanted to make sure readers didn't think nimbess would handle such things. So I could see something like: "translating NB application-specific network related resources into network oriented...". That doesn't sound great either. I guess the original sentence does imply just network so maybe it is OK. I am fine if it makes sense as is and can resolve this.